tests/test_logging.py

```diff
@@ -1,10 +1,9 @@
-import pytest
 import logging
 
 
 def test_logging(wandb_init):
     root_logger = logging.getLogger()
-    root_logger.setLevel("DEBUG")
+    root_logger.setLevel("INFO")
```

Comment:
If set to `DEBUG`, this will potentially add two more entries to the root log: the stuff from WB-6348 and also git complaining about some random CYGWIN check on Windows.